bitnami-services/how-to-guides/index.hbs.md

```diff
@@ -6,4 +6,5 @@ In this section:
 
 - [Configure private registry and VMware Application Catalog integration for Bitnami Services](configure-private-reg-integration.hbs.md)
 - [Obtain credentials for VMware Application Catalog integration](obtain-credentials-for-vac-integration.hbs.md)
+- [Upgrading to TKR 1.26 or newer](upgrading-tkr.hbs.md)
 - [Troubleshoot Bitnami Services](troubleshooting.hbs.md)
```

Comment:
**thought** Might also be good to add a quick section to the troubleshooting doc which links to the Upgrading to TKR doc.

Reply:
Done, please take a look!